projects/1.18/assets/enderite-lieonlion/lolenderite/lang/zh_cn.json

```
@@ -0,0 +1,35 @@
+{
+	"item.lolenderite.enderite_ingot": "末影合金锭",
+	"item.lolenderite.raw_enderite": "原生末影合金",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

这是末影合金的合成材料吗，如果是，建议：

```suggestion
	"item.lolenderite.raw_enderite": "粗末影合金",
```
原文：**Raw Enderite**